src/main/java/application/data/BankAccount.java

```diff
@@ -55,8 +56,18 @@ private BigDecimal getNumberBankAccount() {
 
     /** --------------------- Wlasciciel -------------------------- */
 
-    private UserData getAccountHolder() {
-        return this.accountHolder;
+    public boolean hasAccountHolder(){
+        // it gives information about existence of account holder in current bank account, concurrently avoiding getter
+        return this.accountHolder != null;
+    }
+
+    public void setAccountHolder(){
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Czy chcesz dodac dane wlasciciela tego konta? (T/N): ");
+        if (scanner.nextLine().equalsIgnoreCase("T")) {
+            accountHolder = new UserData();
+            accountHolder.enterUserData();
+        } else System.out.println("Podpowiedz: Dane mozesz wpisac w kazdej chwili wybierajac opcje w menu startowym.");
```

Comment:
Czy setter nie powinien przyjmować właściciela jako parametr a nie sam to wczytywać?

Reply:
Tak właściwie to błąd nazewnictwa, dobra uwaga. Miało być enterAccountHolder...